Libraries/Modal/Modal.js

```diff
@@ -153,16 +154,19 @@ class Modal extends React.Component {
   static contextTypes = {
     rootTag: PropTypes.number,
   };
+
   constructor(props: Object) {
     super(props);
     Modal._confirmProps(props);
   }
+
   componentWillReceiveProps(nextProps: Object) {
     Modal._confirmProps(nextProps);
   }
+
   static _confirmProps(props: Object) {
     if (props.presentationStyle && props.presentationStyle !== 'overFullScreen' && props.transparent) {
-      console.warn(`Modal with '${props.presentationStyle}' presentation style and 'transparent' value is not supported.`);
+      warning(false, `Modal with '${props.presentationStyle}' presentation style and 'transparent' value is not supported.`);
```

Comment:
Seems to me that throughout the code, using `warning(false, ...)` is preferable. See links below:

https://github.com/facebook/react-native/blob/e13b9c6e49480e8262df06b7c1e99caab74e801f/Libraries/Components/TextInput/TextInput.js#L644

https://github.com/facebook/react-native/blob/5257c35d05578bab0233254da81f1216f461f18d/Libraries/Utilities/BackAndroid.js#L35

https://github.com/facebook/react-native/blob/9b4a644fec94422c2976f11be0ccf66f2715d9bd/Libraries/Network/XMLHttpRequest.js#L185